src/components/common/Stepper.vue

```diff
@@ -135,7 +135,7 @@ export default class Stepper extends Vue {
   }
 
   .v-btn:before {
-    background-color: #1976d2;
+    background-color: $app-blue;
```

Comment:
I was skeptical about changing this colour (and below), but the stepper looked the same before and after -- I think some other overrides are actually setting the colour here (and/or below). This can be updated in a future ticket if needed.